packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -324,3 +347,33 @@ export interface IContextualMenuItem {
    */
   [propertyName: string]: any;
 }
+
+export interface IContextualMenuSection extends React.Props<ContextualMenu> {
+
+  /**
+   * The items to include inside the section.
+   */
+  items: IContextualMenuItem[];
```

Comment:
Since items is of type IContextualMenuItem, it could also have a menu section as a child. Is that supported?

Reply:
I think that it might be better to model this after GroupedList and the way that it handles rendering a section. Instead of having IContextualMenuSection containing a list of items, it contains the start index of the items that it should use.

Reply:
Hmm, I don't think I like that approach. Part of the intent of this change is to prevent users from  having to specify a flat list of items to render.